src/app/app-routing.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { MyBlogComponent } from '../app/blog/components/my-blog/my-blog.component';
+import { PostFormComponent } from '../app/shared/components/create-post-form/create-post-form.component';
+import { UserComponent } from './user/user.component';
+
+const routes: Routes = [
+  {
+    path: 'home',
+    component: MyBlogComponent,
+  },
+  {
+    path: 'new-article',
+    component: PostFormComponent,
+  },
+  {
+    path: 'user',
+    component: UserComponent,
+  },
+  {
+    path: '**',
+    redirectTo: 'home',
+  },
+];
```

Comment:
Optional
Better to split it to:
ArticlesModules
UsersModule

Inside ArticlesRoutingModule:
```
const routes: Routes = [
  {
    path: 'articles',
    component: ...ListComponent
    children: [
    {
      path: 'new'
      component: ...FormComponent
    }
    ... others components
    ]
  },
  {
    path: 'new-article',
    component: PostFormComponent,
  },
```